### DISCUSSION TITLE
Detected by akamai

### DISCUSSION THREAD

**Discussion by LimitedTime19** [seleniumbase/SeleniumBase#2269] [Q&A] [answered]
Hi, seleniumbase is working well against cloudflare. However, I found that seleniumbase is not working at all for akamai. The difference between cloudflare and akamai is that akamai frequently checks for chromedriver more than cloudflare which is a big problem. Hoping to see if you know a solution to websites that have bot protects that checks at a random interval. For example one website that I have issues with is nike.com/login. Thanks in advanced!

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/166

There are some SeleniumBase methods you can try though:

```python
driver.reconnect(timeout)
driver.uc_open_with_reconnect(url, reconnect_time=None)
driver.uc_click(selector)
```

**Reply by LimitedTime19** [reply]
Thanks, does driver.reconnect do a driver disconnect and then reconnect at the time I want?

**Reply by mdmintz** [reply]
Yes, it disconnects, and then reconnects after the number of seconds specified has been reached.

**Reply by LimitedTime19** [reply]
Thanks, one last question is, how do I properly pick the right timeout to ensure the page isn't scanning for bot? what's the default timeout?

**Reply by mdmintz** [reply]
The default timeout is set here:
https://github.com/seleniumbase/SeleniumBase/blob/f9a31a39c5544cbb5d4d290dcec9634fea14cddb/seleniumbase/fixtures/constants.py#L365
(Currently at 2.27 seconds, and may change.)
Websites are different, and may change how often they look for traces of automation.
